functions/paperChannelTenders/src/middlewares/validators.ts

```diff
@@ -1,8 +1,39 @@
-import validatorMiddleware from '@middy/validator';
-import { transpileSchema } from '@middy/validator/transpile';
-import { schemaEventGetTenders } from '../types/schema-request-types';
+import {
+  CostEventSchema,
+  CostsEventSchema, DeliveryDriversEventSchema,
+  Event,
+  EventSchema,
+  TenderActiveEventSchema,
+  TendersEventSchema,
+} from '../types/schema-request-types';
 
+const validatorEvent = (event: unknown): Event => {
+  const parsedEvent = EventSchema.safeParse(event);
```

Comment:
E' inutile validare l'evento se poi sotto si applicano tutti i validatori in cascata, togliere anche l'if

Reply:
resolved

Reply:
resolved